tests/integration_tests/test_ingest_component_all_funds.py

```diff
@@ -128,7 +128,17 @@ def test_multiple_rounds_multiple_funds_end_to_end(
     expected_submission_ref["SubmissionDate"] = expected_submission_ref["SubmissionDate"].dt.round("min")
     df_dict["SubmissionRef"]["SubmissionDate"] = df_dict["SubmissionRef"]["SubmissionDate"].dt.round("min")
 
-    assert_frame_equal(expected_submission_ref, df_dict["SubmissionRef"])
+    # Check if the submission was made within the past minute
+    for expected, actual in zip(
+        expected_submission_ref["SubmissionDate"], df_dict["SubmissionRef"]["SubmissionDate"], strict=False
+    ):
+        assert abs((expected - actual).total_seconds()) < 60, f"SubmissionDates don't match: {expected} != {actual}"
+
+    # Check other columns
+    assert_frame_equal(
+        expected_submission_ref.drop(columns=["SubmissionDate"]),
+        df_dict["SubmissionRef"].drop(columns=["SubmissionDate"]),
+    )
```

Comment:
Please can you refer to Will's PR which puts a proper fix for this test in place :)

Reply:
Does it mean to wait until his branch is merged in to main and then rebase my branch on main where this test is fixed, or to install freezegun and then Will to revert 90% of his PR?

Reply:
Will's PR is a better fix for the broken test than the implementation you've currently provided, IMO.

You can either review Will's PR and get it merged, or you could close his PR, cherry-pick the commit onto your branch and drop this slightly-hacky fix.

I think either is fine.

Reply:
Done, thank you!